gatling-engine/src/main/scala/io/snice/gatling/Engine.scala

```diff
@@ -2,21 +2,59 @@ package io.snice.gatling
 
 import io.gatling.app.Gatling
 import io.gatling.core.config.GatlingPropertiesBuilder
-import io.snice.gatling.simulations.{Ss7AttachmentSimulation, UlrSimulation}
+import io.snice.gatling.simulations.{Ss7AttachmentSimulation, StartDiameterStack}
+
+import scala.sys.exit
 
 object Engine extends App {
 
-  val props = new GatlingPropertiesBuilder()
+  override def main(args: Array[String]): Unit = {
+    val usage = """
+      Usage: [--operation operation_name]
+    """
 
-  Option(System.getProperty("simulation")) match {
-    case Some(simulation) => props.simulationClass(simulation)
-    case None => props.simulationClass(classOf[Ss7AttachmentSimulation].getName)
-  }
+    def getOptions(map : Map[String, String], list: List[String]) : Map[String, String] = {
```

Comment:
wondering why we don't make the mocking as part of the Simulation itself. you may have an `Ss7AttachmentSimulationWithMock` that starts also the diameter stack